.github/workflows/easy-cp.yml

```diff
@@ -29,15 +29,9 @@ jobs:
       - name: Get Cherry Pick PR
         run: |
           echo "COMMIT_SHA=$(echo ${{ github.event.pull_request.merge_commit_sha }})" >> $GITHUB_ENV
-      - name: Checkout Flutter Repo
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
-        with:
-          repository: flutteractionsbot/flutter
-          token: ${{ secrets.FLUTTERACTIONSBOT_CP_TOKEN }}
```

Comment:
If the issue is with the token, can we remove the token here? This step shouldn't need any secrets to run as the repos are public.

Reply:
yeah this part is deleted in this PR.

The token is used in the step below where `git push` requires it. The token is required for push if we don't want to set up ssh keys.

Reply:
What I'm saying is revert this diff, and remove the token parameter to the checkout action. I'm not sure I understand why we need to run `git clone` locally here as there may be other flakes that surface.

Reply:
Thanks for clarifying! in order to revert the diff, I did more research on why actions/checkout have token/credential issue, and my current understanding is that the `persist-credentials: false` field is needed (added this field in the most recent patch). This ensures that the checkout step uses default token, but does not override the auth token used in the `git push` step below.

Yeah previously I removed actions/checkout entirely to remove its influence on credentials. The most recent patch adds it back.